frontend/src/Components/AlgInputs/AlgInputsCBA.js

```diff
@@ -255,6 +257,50 @@ class AlgInputsCBA extends Component {
         this.setState({ openAlert: false });
     };
 
+    downloadCbaOpen = () => {
+        this.setState({ openDownloadCbaPopUp: true });
+    };
+
+    downloadCbaClose = () => {
+        this.setState({ openDownloadCbaPopUp: false });
+    };
+
+    downloadCba = async () => {
+        try {
+            const profileMatch = this.state.profileData.filter(
+                (profile) => profile.config_name === this.state.profileName
+            );
+            const countyMatch = profileMatch.map(
+                (profile) => profile["choice"]
+            );
+            const params = {
+                load_profile: this.state.profileName,
+                county: countyMatch[0],
+            };
+            const FileDownload = require('js-file-download');
```

Comment:
Instead of the dynamic require, can we import at the top of the file?

Reply:
will do thanks!

Reply:
Updated